### DISCUSSION TITLE
Performance - 2

### DISCUSSION THREAD

**Discussion by changjurhee** [changjurhee/AONE#14] [Polls]
## Quality Attributes

Type | Description | QA Scenario (Stimulus / Source of stiulus / Artifact / Environment / Response / Response measure) | Tatics
 -- | -- | -- | --
Performance | Provide high quality low latency with strategies for dealing with packet or data loss and jitter. | Jitter is introduced by the gateway, so the call should operate without a delay of more than one second.<br>- Jitter occurs<br>- Gateway<br>- Call service<br>- Runtime<br>- adjust buffer size.<br>- end-to-end Latency is less than 1 (?) second, frame rate is dropped less than 10 (?)  %. resolution content bitrate   |

**Comment by codekistone** [comment]
reduce overhead - If the bandwidth is insufficient, reduce the transmission quality of the transmission video or make it operate as voice only so that there is no disconnection of the call connection.

**Comment by changjurhee** [comment]
- frame rate drop 허용 % ? 
- latency <= ?
